redis-lettuce/src/main/java/io/micronaut/configuration/lettuce/AbstractRedisConfiguration.java

```diff
@@ -16,7 +16,6 @@
 package io.micronaut.configuration.lettuce;
 
 import io.lettuce.core.RedisURI;
-import io.lettuce.core.resource.ClientResources;
```

Comment:
had to remove this as CI was complaining that it is not used (it is used in the javadoc though)